### PR TITLE
reset + rename some default params

### PR DESCRIPTION
**Pull request by kpeluso**
Diego researched what parameter values should be and I reset them. I also renamed some parameters to more easily-usable and predictable names in the process.

### CODE REVIEW

**Review by dmikey — APPROVED** [review]

File: x/emissions/proto/emissions/state/v1/tx.proto [hunk 1]
```diff
@@ -27,74 +28,100 @@ service Msg {
...
+    (cosmos_proto.scalar) = "cosmos.Uint",
     (gogoproto.customtype) = "cosmossdk.io/math.Uint",
-    (gogoproto.nullable)   = false,
+    (gogoproto.nullable) = false,
     (amino.dont_omitempty) = true
-  ]; // total unmet demand for a topic < this => don't run inference solicatation or weight-adjustment
-  repeated uint64 max_topics_per_block = 5; // max number of topics to run cadence for per block
+  ]; // total unmet demand for a topic < this => don't run inference
+     // solicatation or weight-adjustment
+  repeated uint64 max_topics_per_block =
```
Comment (relyt29) [inline x/emissions/proto/emissions/state/v1/tx.proto:96]: why is the formatting in this file all weird, is that necessary

Comment (relyt29) [reply x/emissions/proto/emissions/state/v1/tx.proto:96]: the excess newlines, i mean

**Comment by relyt29** [comment]
> looks ok to me pending you fix whatever broken test that is

seems like it might be demand_test.go:478

File: x/emissions/proto/emissions/state/v1/tx.proto (hunk 1 above)
Comment (kpeluso) [reply x/emissions/proto/emissions/state/v1/tx.proto:96]: linter just did that idk. all around that file

File: x/emissions/proto/emissions/state/v1/tx.proto [hunk 2]
```diff
@@ -27,74 +28,100 @@ service Msg {
...
+      5; // max number of topics to run cadence for per block
   repeated string min_request_unmet_demand = 7 [
```
Comment (xmariachi) [inline x/emissions/proto/emissions/state/v1/tx.proto:98]: No field number 6 here. Is that jump done for a reason? It wont affect Proto compiler but it just looks better.

**Review by xmariachi — COMMENTED** [review]
Comment on proto seq

File: x/emissions/proto/emissions/state/v1/tx.proto (hunk 2 above)
Comment (kpeluso) [reply x/emissions/proto/emissions/state/v1/tx.proto:98]: true

Comment (kpeluso) [reply x/emissions/proto/emissions/state/v1/tx.proto:98]: fixed in #56

**Pull request merged**